src/editors/playgroundController.ts

```diff
@@ -125,4 +119,15 @@ export default class PlaygroundController {
       resolve(true);
     });
   }
+
+  public deactivate(): void {
+    if (this._connectionController) {
+      this._connectionController.removeEventListener(
+        DataServiceEventTypes.ACTIVE_CONNECTION_CHANGED,
+        () => {
+          this.showActiveConnectionInPlayground('Playground disconnected');
```

Comment:
I'm not sure I follow the expected behavior here, when the mongodb app deactivates we want to show this message?

Reply:
Yeah, I just added it to check when deactivation happens, but have never seen this message actually. What do you think is better here? Empty function?

Reply:
I think empty function is probably better just so we preserve what is in the `output`.